src/config.rs

```diff
@@ -38,11 +38,14 @@ pub struct Config {
     // Bitwarden connection config
     bitwarden_url: String,
     bitwarden_admin_token: String,
+    bitwarden_root_cert: Option<String>,
     // LDAP Connection config
     ldap_host: String,
     ldap_scheme: Option<String>,
     ldap_ssl: Option<bool>,
     ldap_port: Option<u16>,
+	// LDAP skip tls verify
```

Comment:
Either fix the indent (`rustfmt`) or you can remove this. I think that the variable name you picked is really quite descriptive. 👍

Reply:
Comment removed.